src/data/activities.json

```diff
@@ -10,7 +10,7 @@
   "no-header": {
     "ID": "no-header",
     "roomID": "roomID",
-    "text": "text",
+    "text": "no header text",
```

Comment:
I would prefer to have "Lorem Ipsum" text whenever possible

Reply:
not on this one. I want to emphasize that this is a no header text. I did it for the other ones though